config/variables_test.go

```diff
@@ -8,54 +8,6 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-func TestFormatPrompt(t *testing.T) {
-	t.Parallel()
-
-	variable := Variable{Name: "foo"}
-	options := &BoilerplateOptions{}
-
-	actual := formatPrompt(variable, options)
-	expected := "Enter a value for variable 'foo'"
-
-	assert.Equal(t, expected, actual)
-}
-
-func TestFormatPromptWithDefault(t *testing.T) {
-	t.Parallel()
-
-	variable := Variable{Name: "foo", Default: "bar"}
-	options := &BoilerplateOptions{}
-
-	actual := formatPrompt(variable, options)
-	expected := "Enter a value for variable 'foo' (default: 'bar')"
-
-	assert.Equal(t, expected, actual)
-}
-
-func TestFormatPromptWithCustomPrompt(t *testing.T) {
-	t.Parallel()
-
-	variable := Variable{Name: "foo", Prompt: "custom"}
-	options := &BoilerplateOptions{}
-
-	actual := formatPrompt(variable, options)
-	expected := "custom"
-
-	assert.Equal(t, expected, actual)
-}
-
-func TestFormatPromptWithCustomPromptAndDefault(t *testing.T) {
-	t.Parallel()
-
-	variable := Variable{Name: "foo", Prompt: "custom", Default: "bar"}
-	options := &BoilerplateOptions{}
-
-	actual := formatPrompt(variable, options)
-	expected := "custom (default: 'bar')"
-
-	assert.Equal(t, expected, actual)
-}
-
```

Comment:
Your reasoning makes sense. In fact, I was listening to a [podcast on effective unit testing](http://www.se-radio.net/2016/05/se-radio-episode-256-jay-fields-on-working-effectively-with-unit-tests/) and the guest actively encouraged the pruning of unit tests that had outlived their usefulness. So, sounds like you made the right call!